man/systemd-system.conf.xml

```diff
@@ -370,6 +370,18 @@
         100ms.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DefaultDeviceTimeoutSec=</varname></term>
+
+        <listitem><para>Configures the default timeout for waiting for devices. It can only be changed per a device via
+        the <varname>x-systemd.device-timeout=</varname> option in <filename>/etc/fstab</filename> and
+        <filename>/etc/crypttab</filename> (see
+        <citerefentry><refentrytitle>systemd.mount</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+        <citerefentry><refentrytitle>crypttab</refentrytitle><manvolnum>5</manvolnum></citerefentry>). It defaults to
+        90s.
```

Comment:
that's not true. It can also be changed by doing a .device.d/ drop-in which sets JobTimeoutSec=

Reply:
also, drop the "only". That's an unnecessary info that restricts what we can change later on.